tests/framework/lib/framework.c

```diff
@@ -136,7 +137,7 @@ mtf_main(int argc, char **argv, struct mtf_test_coll_info *tci)
             stderr,
             "%s: hse_init failed: %s\n",
             progname,
-            merr_strinfo(err, errbuf, sizeof(errbuf), NULL));
+            merr_strinfo(err, errbuf, sizeof(errbuf), (merr_stringify)(err_ctx_strerror), NULL));
```

Comment:
Change `err_ctx_strerror` to take an `int` arg instead of `enum` so that we don't have to cast everywhere?

Reply:
I was thinking about that if you guys really don't like the cast.